datafusion/sqllogictest/test_files/struct.slt

```diff
@@ -373,3 +373,52 @@ You reached the bottom!
 
 statement ok
 drop view complex_view;
+
+# Test row alias
+
+query ?
+select row('a', 'b');
```

Comment:
BTW there is pretty sweet struct literal syntax like `{ c1: 'a', c2: 'b'}` as well

```sql
> select { c1: 'a', c2: 'b'};
+---------------------------------------------------------+
| named_struct(Utf8("c1"),Utf8("a"),Utf8("c2"),Utf8("b")) |
+---------------------------------------------------------+
| {c1: a, c2: b}                                          |
+---------------------------------------------------------+
1 row(s) fetched.
Elapsed 0.002 seconds.
```

Reply:
In DuckDB, they don't have column name. Stored like a tuple.

```
D select row('red', 1);
┌──────────────────────────┐
│    main.row('red', 1)    │
│ struct(varchar, integer) │
├──────────────────────────┤
│ (red, 1)                 │
└──────────────────────────┘
```